sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseGridDetailed.java

```diff
@@ -94,5 +118,9 @@ protected void initColumns() {
 					value.getBirthdateYYYY(),
 					I18nProperties.getUserLanguage()),
 			new TextRenderer());
+
+		((Column<CaseIndexDetailedDto, Date>) getColumn(CaseIndexDetailedDto.SYMPTOMS_ONSET_DATE))
+			.setRenderer(new DateRenderer(DateFormatHelper.getDateFormat()))
```

Comment:
use setCaption here together with `I18nProperties.getPrefixCaption` to get the onsetDate caption from Symptoms